posts/2021-03-22-ldap-kerberos-21004beta.adoc

```diff
@@ -96,7 +96,7 @@ The following example shows how to configure an LDAP Registry using a keytab and
 <ldapRegistry id="LDAP" realm="SampleLdapADRealm" host="ldap_hostname" port="389" ignoreCase="true" baseDN="DC=example,DC=com" bindAuthMechanism="GSSAPI" krb5Principal="user1@EXAMPLE.COM" ldapType="Custom" />
 ----
 
-For more information on `LdapRegistry`, see the link:https://openliberty.io/docs/latest/reference/feature/ldapRegistry-3.0.html[LDAP User Registry documentation].
```

Comment:
this bit was correct, it's the next sentence that needs change

Reply:
Done